shell/tabry_fish.fish

```diff
@@ -0,0 +1,108 @@
+
+set -x _tabry_rs_import_path "$_tabry_rs_import_path:~/.local/share/tabry"
```

Comment:
I don't yet expand ~ in the rust code so the default isn't working.
I'll wrap up testing and reviewing tomorrow

Reply:
Ah, okay, I just copied this from the `tabry_bash.sh` file

Reply:
Do you want me to leave this in so it just starts working when tabry supports it?

Reply:
I can also just do something like: ```:$HOME/.local/share/tabry"``` if that's easier?

Reply:
I pushed a change to expand ~/ in master. See https://github.com/evanbattaglia/tabry-rs/pull/2#issuecomment-2329659143 -- do you know if there's any easy way to avoid running tabry 5 times?

Reply:
Ok, yeah, I am completely switching up the approach, and I actually think I can  support only matching directories now... still working on it at the moment, and will push it to this PR when I'm done